src/main/java/slimeknights/tconstruct/library/recipe/tinkerstation/ITinkerStationRecipe.java

```diff
@@ -40,12 +41,11 @@ default ItemStack assemble(ITinkerStationContainer inv) {
 
   /**
    * Gets the recipe result, or an object containing an error message if the recipe matches but cannot be applied.
-   * TODO 1.20: switch return type to {@code RecipeResult<LazyToolStack>}
    * @return Validated result
    */
-  default RecipeResult<ItemStack> getValidatedResult(ITinkerStationContainer inv) {
-    ItemStack result = assemble(inv);
-    if (result.isEmpty()) {
+  default RecipeResult<LazyToolStack> getValidatedResult(ITinkerStationContainer inv) {
+    LazyToolStack result = LazyToolStack.from(assemble(inv));
+    if (result.getStack().isEmpty()) {
       return RecipeResult.pass();
     }
     return RecipeResult.success(result);
```

Comment:
Probably better to do
```java
ItemStack result = assemble(inv);
if (result.isEmpty()) {
  return RecipeResult.pass()
}
return RecipeResult.success(LazyToolStack.from(result));
```
No sense creating a lazy tool stack if we are not returning it.